packages/nextjs/package.json

```diff
@@ -71,8 +71,7 @@
     "fix": "eslint . --format stylish --fix",
     "lint": "eslint . --format stylish",
     "test": "yarn test:unit",
-    "test:all": "run-s test:unit test:integration test:build",
```

Comment:
Were these tests unused because we never called `test:all` in CI?

Reply:
yes we only called `test` and `test:integration`